apiserver/facades/agent/uniter/uniter.go

```diff
@@ -1645,7 +1681,18 @@ func (u *UniterAPI) NetworkInfo(args params.NetworkInfoParams) (params.NetworkIn
 		}
 	}
 
-	networkInfos := machine.GetNetworkInfoForSpaces(spaces)
+	// Don't get the network info for the default space if there was an error
```

Comment:
why?
also, does it necessarily hold true that it's the default space for cross-model?